v2/spiffebundle/bundle.go

```diff
@@ -0,0 +1,43 @@
+package spiffebundle
+
+import (
+	"crypto/x509"
+	"io"
+
+	"gopkg.in/square/go-jose.v2"
+)
+
+// Bundle is a SPIFFE bundle.
+type Bundle struct {
+	jose.JSONWebKeySet
```

Comment:
I'm not sure how I feel about exposing the world to go-jose through this library. IMO this package should be heavily opinionated, and I worry about what people may use/abuse under the covers. I think I'd prefer to see us hide this away as an implementation detail, and instead expose only what makes sense for our use cases.

One basic thing that comes to mind is going from a *Bundle to []*x509.Certificate via something like `*Bundle.X509CAs()`

Reply:
That's a fair point. I'll see what I can do to hide it.